docs/SRS-Volere/SRS.tex

```diff
@@ -348,9 +348,17 @@ \subsection{Cultural Requirements}
 
 \section{Compliance Requirements}
 \subsection{Legal Requirements}
-\lips
+\begin{itemize}
+    \item The application is to strictly abide by all regulations governing
```

Comment:
Requirements should have an identifier associated with them to aid in traceability

Reply:
Thanks, added identifiers